Kernel/Process.cpp

```diff
@@ -1803,6 +1804,34 @@ int Process::sys$mkdir(const char* pathname, mode_t mode)
     return VFS::the().mkdir(StringView(pathname, pathname_length), mode & ~umask(), current_directory());
 }
 
+int Process::sys$realpath(const char* pathname, char* buffer, ssize_t size)
+{
+    if (size < 0)
+        return -EINVAL;
+    if (!validate_read_str(pathname))
+        return -EFAULT;
+
+    size_t pathname_length = strlen(pathname);
+    if (pathname_length == 0)
+        return -EINVAL;
+    if (pathname_length >= 255)
```

Comment:
We should change both this and `mkdir` to use `PATH_MAX` instead.

Reply:
Now I pass the buffer size to the function, since it seems `PATH_MAX` cannot be accessed to from `Process.cpp`. Currently, I've left `mkdir` as it is.